assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -896,9 +899,28 @@ export default Component.extend({
     next(this.reStickScrollIfNeeded.bind(this));
   },
 
+  @discourseComputed()
+  canQuote() {
+    if (this.chatChannel.chatable_type === "DirectMessageChannel") {
```

Comment:
We've got this string littered many places (I've contributed to it too probably) but maybe we can start using `CHATABLE_TYPES.directMessageChannel`

Reply:
Yeah I will do that in a separate PR, might even make sense to export a proper model for chatChannel and have shortcut methods like `isDirectMessage` like the ruby chat_channel model.

Reply:
Thank you <3